AppCenter/AppCenterTests/MSDeviceTrackerTests.m

```diff
@@ -351,6 +351,39 @@ - (void)testWrapperSdk {
   XCTAssertEqual(device.wrapperSdkVersion, wrapperSdk.wrapperSdkVersion);
 }
 
+- (void)testCountryCode {
```

Comment:
This test case covers macOS/tvOS, it would be good to have another test case for calling `setCountryCode` on iOS and verify it shouldn't override the country code.

Reply:
We could hide the API in the iOS headers so it's a macOS only API, too?

Reply:
Let's trash that idea, we have wifi only iPads, too.

Reply:
Ok, I've added a test for this in c63bc36dae57f528247bebf7e1955a003b8c49e6.